ios/RNSScreenStack.mm

```diff
@@ -71,47 +94,74 @@ @implementation RNSPanGestureRecognizer
 @implementation RNSScreenStackView {
   UINavigationController *_controller;
   NSMutableArray<RNSScreenView *> *_reactSubviews;
-  __weak RNSScreenStackManager *_manager;
-  BOOL _hasLayout;
   BOOL _invalidated;
+  BOOL _isFullWidthSwiping;
   UIPercentDrivenInteractiveTransition *_interactionController;
+  BOOL _hasLayout;
+  __weak RNSScreenStackManager *_manager;
+#ifdef RN_FABRIC_ENABLED
   BOOL _updateScheduled;
-  BOOL _isFullWidthSwiping;
+  UIView *_snapshot;
+#else
+  BOOL _updateScheduled;
```

Comment:
Why is it in both options?

Reply:
[aad05ee](https://github.com/software-mansion/react-native-screens/pull/1416/commits/aad05ee04facd453c70b19181e7c2b60dc0d4e68)